apps/web/app/(authenticated)/(app)/app/settings/root-keys/page.tsx

```diff
@@ -22,27 +22,11 @@ export default async function SettingsKeysPage(props: { params: { apiId: string
     return redirect("/onboarding");
   }
 
-  const found = await db.query.keys.findMany({
+  const keys = await db.query.keys.findMany({
     where: eq(schema.keys.forWorkspaceId, workspace.id),
     limit: 100,
   });
 
-  const keys: Key[] = [];
-  const expired: Key[] = [];
-
-  for (const k of found) {
-    if (k.expires && k.expires.getTime() < Date.now()) {
-      expired.push(k);
-    }
-    // remove temp keys from the list of keys.
-    if (!k.expires) {
-      keys.push(k);
-    }
-  }
-  if (expired.length > 0) {
-    await Promise.all(expired.map((k) => db.delete(schema.keys).where(eq(schema.keys.id, k.id))));
-  }
-
```

Comment:
Do we want to actually show the keys we generate from tRPC / Dashboard? By default our root keys don't expire but the ones we create do.

Reply:
reverted in next commit